_episodes/01-learner-profile.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Why should I take this course?"
+teaching: 8
+exercises: 7
+questions:
+- "Why would I be taking this course?"
+- "What can I expect from this course?"
+objectives:
+- "Identify performance bottlenecks of running LAMMPS in a HPC cluster"
+- "Be enable to use hardware architecture support at run-time using KOKKOS"
```

Comment:
Don't introduce Kokkos at this stage, just leave it at `How to enable hardware architecture support in LAMMPS`